app/javascript/components/index.js

```diff
@@ -1,4 +1,5 @@
 import MappingWizardContainer from '../react/screens/App/Overview/screens/MappingWizard';
+// import MappingWizardGeneralStepContainer from '../react/screens/App/Overview/screens/MappingWizard/components/MappingWizardGeneralStep';
```

Comment:
We should clean up these commented-out lines before merging, if you weren't doing so already.

Reply:
Yes, I left the comments in there to basically underline the fact that `componentRegistry` (and the related code like this one) does not work with Redux form.

I will be removing them.

Reply:
Addressed in d3da630

Reply:
i am fine w/ removing those comments for now ;)